webapp/tutorials/views.py

```diff
@@ -0,0 +1,70 @@
+from os import getenv
+
+import flask
+import talisker.requests
+
+from canonicalwebteam.discourse import DiscourseAPI, TutorialParser, Tutorials
+
+DISCOURSE_API_KEY = getenv("DISCOURSE_API_KEY")
+DISCOURSE_API_USERNAME = getenv("DISCOURSE_API_USERNAME")
```

Comment:
you need to the the env keys in `konf/site.yaml` https://github.com/canonical-web-and-design/ubuntu.com/blob/main/konf/site.yaml#L16

Reply:
Totally forgot to add it, thanks!